client/components/web-preview/index.jsx

```diff
@@ -47,8 +56,13 @@ const WebPreview = React.createClass( {
 	getDefaultProps() {
 		return {
 			showExternal: true,
```

Comment:
That's a good point. I plan to use this in a wrapper which does set `showExternal` to false (#4140) but I suppose there's never a time when `previewMarkup` is set that we'd want that button to be there.